shared/provision/select-other-device/index.tsx

```diff
@@ -11,9 +11,6 @@ type Props = {
   onBack: () => void
 }
 
-// needs to be an invalid device name
-const resetSentry = '!reset'
```

Comment:
Fortunately you've realized your error. The first revision of the fix isn't idiomatic TypeScript, and further usage of this pattern can lead to more reliability issues. Not using the name field at all is the perfect approach. Thanks for the fast response.